app/main/helpers/suppliers.py

```diff
@@ -0,0 +1,24 @@
+import os
+import json
+
+
+def load_countries():
+    helpers_path = os.path.abspath(os.path.dirname(__file__))
+    countryfile = os.path.join(helpers_path, '../../static/location-autocomplete-canonical-list.json')
+    with open(countryfile) as f:
+        return json.load(f)
+
+
+def get_country_name_from_country_code(country_code):
+    if country_code and country_code != 'gb':
+        potential_names = [country[0] for country in COUNTRY_LIST if country[1] == country_code]
+        return potential_names[0] if potential_names else ''
+    elif country_code:
+        # We need to support the old code style ('gb') until after the existing data has updated.
```

Comment:
after the existing data has updated?

What existing data? What process will update it? Is this something that will happen naturally or are running a migration later?

Reply:
This is referring to the existing country code data be have for UK suppliers. It'll get updated by a migration. I'll make the comment more explicit.